Sources/StreamChatUI/ChatChannelList/ChatChannelAvatarView_Tests.swift

```diff
@@ -24,14 +24,14 @@ class ChatChannelAvatarView_Tests: XCTestCase {
     func test_emptyAppearance() {
         let view = ChatChannelAvatarView().withoutAutoresizingMaskConstraints
         view.addSizeConstraints()
-        AssertSnapshot(view, variants: [.small])
+        AssertSnapshot(view, variants: [.smallAndDark])
```

Comment:
This doesn't seem to work well. Here we have a view that's content-size agnostic and we want to test only different user interface style values. So, from the API point of view, this should look something like:
```
AssertSnapshot(view, variants: .allUserInterfaceStyles)

... or

AssertSnapshot(view, variants: .onlyUserInterfaceStyles)

... or we can even flip the logic around and specify the snapshots you don't want:

AssertSnapshot(view, variants: .withoutContentSizeChanges)
```

I'd assume it will create 2 snapshots (just 1 on iOS <12) with the default content size.

For views where the content size matters, the API could look like just:
```
AssertSnapshot(view) 
```
which means "snapshot the full matrix of options".

Reply:
Actually, in the case of ChatAvatarView, we are testing an UIImage, so the dark and light traits are not very useful (I guess? 🤔) Unless we have border colours etc, in that case testing the default `.allUserInterfaceStyles` is actually useful. That is why didn't use the default arguments here. So maybe we don't need here to specify the variant at all and just use the default which tests all interface styles. 

If not, we can actually create variants without the dark and light trait, and just name them "small" and "large" or "default".

But currently `AssertSnapshot` for viewControllers use `.all` by default. When testing views, `AssertSnapshot` uses `.allUserInterfaceStyles`. Do you want me, in this case, to not provide any custom argument and just use the `.allUserInterfaceStyles` ?

Reply:
> Actually, in the case of ChatAvatarView, we are testing an UIImage, so the dark and light traits are not very useful (I guess? 🤔) 

Correct, there's nothing that should react to interface style changes in `ChatAvatarView` but, for example, `ChatChannelAvatarView` has the online indicator which background changes based on the style.

On the other hand, I think it makes sense to test dark/light variant for all views. It will help to prevent regressions when we introduced different appearances by accident.

I don't think it's a good idea to introduce different default values for view and view controllers. It will be confusing. So, what if we did it like this:
```
AssertSnapshot(view) 
// 👆 by default creates the full matrix:
// small + dark
// large + light
// xxl + dark

AssertSnapshot(view, variants: . onlyUserInterfaceStyles) 
//  👆creates only two snaphots (light + dark) with default content size
```

Reply:
@VojtaStavik Changes done ✅